openmdao/recorders/case.py

```diff
@@ -448,28 +448,26 @@ def list_inputs(self,
         inp_vars = {}
         inputs = []
 
-        tags = convert_user_defined_tags_to_set(tags)
-
         if self.inputs is not None:
             for abs_name in self.inputs.absolute_names():
                 if abs_name not in inp_vars:
                     inp_vars[abs_name] = {'value': self.inputs[abs_name]}
```

Comment:
I know this isn't part of the PR, but it seems odd here that we are creating a dict with only one item in it.  Why don't we just store self.inputs[abs_name] in inp_vars[abs_name] directly?

Reply:
cleaned this up in both `list_inputs` and `list_outputs`